CONTRIBUTING.md

```diff
@@ -124,6 +124,49 @@ Instead, write one sentence per line, as this makes it easier to review changes.
    Respond to review comments, potential CI failures, and potential merge conflicts by updating the pull request.
    Always keep the pull request in a mergeable state.
 
+## Triaging an NGI project
+
+When [opening an issue for a new NGI project](https://github.com/ngi-nix/ngipkgs/issues/new?template=project-triaging.yaml), it's recommended to follow the workflow below, which details which information we'd like to know about a project.
+
+1. Navigate to <https://nlnet.nl/project/>.
+   In the search bar, type the project name and look for any related projects.
+
+   ```md
+   - https://nlnet.nl/project/foobar
+   - https://nlnet.nl/project/foobar-core
+   ```
+
+   In the project pages, look for any `website` or `source code` links and open them.
+
+1. In the project's source repository, we'd like to know some information about what `framework` and `dependency management` tools the project uses so we can estimate the time and effort needed to package it. If possible, we'd also like to have links to Nix development environments, if they exist in the repo.
+
+   ```md
+   - Language/Framework: Python/Django
+   - Dependency management: pip
+   - Development envirnment: [default.nix, shell.nix, flake.nix, devenv.nix, ...](<FILE_LINK>)
+   ```
+
+1. In the project's website, look for any tabs or buttons that lead to the documentation. You may also use your favorite search engine and look for `<PROJECT_NAME> documentation`.
+   The most relavant information we want is documentation about building the project from source and examples for using it.
+   If the project has many components, it would be ideal to have this information for each one of them.
+
+   ```md
+   - Usage Examples:
+     - https://foo.bar/docs/quickstart
+   - Build from source/Development:
+     - foobar-cli: https://foo.bar/docs/dev/cli
+     - foobar-mobile: https://foo.bar/docs/dev/mobile
+   ```
```

Comment:
```suggestion
1. In the project's website, look for any tabs or buttons that lead to the documentation. You may also use your favorite search engine and look for `<PROJECT_NAME> documentation`.
   The most important information we need are the instructions for building the project from source and examples for using it.
   If the project has multiple components, it would be ideal to have this information for each one of them.

   ```md
   - Usage Examples:
     - https://foo.bar/docs/quickstart
   - Build from source/Development:
     - foobar-cli: https://foo.bar/docs/dev/cli
     - foobar-mobile: https://foo.bar/docs/dev/mobile
   ```
```